lib/nationbuilder/client.rb

```diff
@@ -127,7 +127,8 @@ def parse_response_body(response)
     error = classify_response_error(response)
     raise error if error
 
-    if response.header['Content-Type'].first != 'application/json'
+    content_type = response.header['Content-Type'].first
+    unless content_type && content_type.include?('application/json')
```

Comment:
Looks like we're now appending the character set which threw off the `Content-Type` check here.